docs/includes/preview-apis-conditions.md

```diff
@@ -0,0 +1 @@
+Preview APIs are subject to change and are not intended for use in a production environment. We recommend that you try them out in test and development environments only. Do not use preview APIs in a production environment or within business-critical documents.<br><br>[!INCLUDE [Information about using preview APIs](../includes/using-preview-apis-host.md)]
```

Comment:
Question: why use HTML breaks instead of two general newlines?

Reply:
if I just do 2 general newlines, that breaks the subsequent content out of the NOTE.

Reply:
I guess this leads back to my question as to why the Outlook preview copy is different. If that was standardized, the Include file could be a >[!NOTE] in and of itself.

Reply:
That's true. I'll see if I can rationalize the content across all of them so they can have the same note.

Reply:
I updated so that they all share the Note but Outlook has an additional note about the preview requirement set.